WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -744,17 +744,28 @@ private void updateSlug(String slug) {
 
     private void updatePassword(String password) {
         EditPostRepository editPostRepository = getEditPostRepository();
-        if (editPostRepository != null) {
-            editPostRepository.updateAsync(postModel -> {
-                postModel.setPassword(password);
-                return true;
-            }, (postModel, result) -> {
-                if (result == UpdatePostResult.Updated.INSTANCE) {
-                    mPasswordTextView.setText(password);
-                }
-                return null;
-            });
-        }
+        if (editPostRepository == null) return;
+
+        String trimmedPassword = password.trim();
+        Boolean isNewPasswordBlank = trimmedPassword.isEmpty();
+        String previousPassword = editPostRepository.getPassword();
+        Boolean isPreviousPasswordBlank = previousPassword.isEmpty() || previousPassword.trim().isEmpty();
```

Comment:
Hi @adityabhaskar would this give the same result hence making the `isEmpty()` redundant? Let me know your thoughts. Thanks 👍 

```suggestion
        Boolean isPreviousPasswordBlank = previousPassword.trim().isEmpty();
```

Reply:
I was just unnecessarily micro optimising—no need to trim if length is already zero :)

Reply:
Awesome! Thanks much @adityabhaskar 🙏